lib/apipie/response_description.rb

```diff
@@ -105,6 +105,15 @@ def initialize(method_description, code, options, scope, block, adapter)
       @response_object.additional_properties ||= options[:additional_properties]
     end
 
+    def response_name
+      suffix = if @code == 200
+                 'response'
+               else
+                 "#{@code}_response"
+               end
+      "#{@method_description.method}_#{@method_description.resource._name}_#{suffix}"
+    end
```

Comment:
This allows us to generate sensible names for response objects defined inline rather than in a param group.